docs/contributing/source/ways-can-contribute/setup-dev-environment.md

```diff
@@ -0,0 +1,69 @@
+# Setup development environment
+
+## Setting up a single node development environment with ``docker-compose``
+
+You can also use [Docker Compose](https://docs.docker.com/compose/) to run all the tests.
+
+```bash
+$ docker-compose build bdb
```

Comment:
isn't this supposed to be `bigchaindb` instead of `bdb`

Reply:
No, it is still `bdb`